eva-accession-release/src/main/java/uk/ac/ebi/eva/accession/release/batch/io/deprecated/DeprecatedVariantMongoReader.java

```diff
@@ -15,41 +15,123 @@
  */
 package uk.ac.ebi.eva.accession.release.batch.io.deprecated;
 
```

Comment:
Substantial rewrite had to be carried out because the criteria for deprecated variant is "RS marked as deprecated in Clustered operations with at least one evidence in SS operations with that RS along with matching assembly/taxonomy"

Reply:
The only change this time around is the criteria for matching taxonomy which we can only get from SS operations...